tests/Feature/PublicSchedule/PublicScheduleTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Tests\Feature\PublicSchedule;
+
+use App\Models\Lecturing;
+use Carbon\Carbon;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class PublicScheduleTest extends TestCase
+{
+    use RefreshDatabase;
+
+    /** @test */
+    public function user_can_see_friday_lecturing_today_in_public_schedule(): void
+    {
+        Carbon::setTestNow('2023-09-16');
+        $lecturing = factory(Lecturing::class)->create([
+            'audience_code' => Lecturing::AUDIENCE_FRIDAY,
+            'date' => '2023-09-16'
+        ]);
+
+        $this->visitRoute('public_schedules.today');
+        $this->seeRouteIs('public_schedules.today');
+        $this->seeText(__('lecturing.friday_lecturer_name'));
+        $this->seeText($lecturing->lecturer_name);
+        $this->seeText(__('lecturing.imam_name'));
+        $this->seeText($lecturing->imam_name);
+        $this->seeText(__('lecturing.muadzin_name'));
+        $this->seeText($lecturing->muadzin_name);
```

Comment:
Tolong tambahkan bagian reset Carbon ini ya di semua testingnya mas.

```suggestion
        $this->seeText($lecturing->muadzin_name);
        Carbon::setTestNow();
```